src/platform/darwin_openssl.c

```diff
@@ -123,7 +123,10 @@ CxPlatTlsVerifyCertificate(
         goto Exit;
     }
 
-    SSLPolicy = SecPolicyCreateSSL(true, SNIString);
+    SSLPolicy =
+        SecPolicyCreateSSL(
+            (CredFlags & QUIC_CREDENTIAL_FLAG_CLIENT) ? FALSE : TRUE,
```

Comment:
This needs to be reversed too. For client side (verifying a server cert) this needs to be true. Also, this is where the merge conflict with main in occuring so it won't run until main is merged back in.